contracts/protocols/polygon/uniswapStaking/helpers.sol

```diff
@@ -97,4 +97,13 @@ abstract contract Helpers is DSMath {
             );
         }
     }
+
+    function isContract(address account) public view returns (bool) {
+        uint32 size;
+        address a = account;
+        assembly {
+            size := extcodesize(a)
+        }
```

Comment:
As we are using inline assembly, I think this function can't be of view type?
Can you try compiling the code?

Reply:
Yeah code is running fine

Reply:
okay